src/Libraries/CoreNodeModels/Input/BasicInteractive.cs

```diff
@@ -44,13 +44,13 @@ public override NodeInputData InputData
             {
                 return new NodeInputData()
                 {
-                    Id = this.GUID.ToString("N"),
+                    Id = this.GUID,
```

Comment:
@gregmarr I think this is node property we do not serialize and used as public getter. They do not affect our serialized DYN for now